src/librustc/session/config.rs

```diff
@@ -413,9 +429,9 @@ top_level_options!(
 
         edition: Edition [TRACKED],
 
-        // The list of crates to consider private when
+        // The crates to consider private when
         // checking leaked private dependency types in public interfaces
-        extern_private: Vec<String> [TRACKED],
+        //extern_private: ExternPrivates [UNTRACKED],
```

Comment:
There are multiple instances of commented out code, could you remove it instead.